pkg/services/servicetest/run.go

```diff
@@ -0,0 +1,65 @@
+package servicetest
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+)
+
+type Runnable interface {
+	Start(context.Context) error
+	Close() error
+}
+
+// Run fails tb if the service fails to start or close.
+func Run(tb testing.TB, s Runnable) {
+	tb.Helper()
+	require.NoError(tb, s.Start(tests.Context(tb)), "service failed to start")
+	tb.Cleanup(func() { assert.NoError(tb, s.Close(), "error closing service") })
+}
+
+// RunHealthy fails tb if the service fails to start, close, is never ready, or is ever unhealthy (based on periodic checks).
+//   - after starting, readiness will always be checked at least once, before closing
+//   - if ever ready, then health will be checked at least once, before closing
+func RunHealthy(tb testing.TB, s services.Service) {
```

Comment:
Is there a sensible unit test for `RunHealthy`?

Reply:
Hmm. Probably not. The use in pkg/loop is likely enough. 

Related: should we add this file to sonar exceptions so we don't mess with coverage stats?

Reply:
Added a test.